src/zerocash/Address.cpp

```diff
@@ -135,16 +121,11 @@ Address Address::CreateNewRandomAddress() {
     getRandBytes(a_sk_bytes, ZC_A_SK_SIZE);
     convertBytesToBytesVector(a_sk_bytes, a_sk);
 
-    AutoSeededRandomPool prng;
-
-    ECIES<ECP>::PrivateKey privateKey;
-    privateKey.Initialize(prng, ASN1::secp256r1());
-
-    std::string encodedPrivateKey;
+    uint256 a_sk_u(a_sk);
```

Comment:
Does the code above this generate an invalid a<sub>sk</sub>? The leading 4 bits should be zero.